scripts/main/photo.js

```diff
@@ -803,13 +803,17 @@ photo.editTags = function (photoIDs) {
 			.map((tag) => tag.trim())
 			.filter((tag) => tag !== "" && tag.indexOf(",") === -1)
 			.sort();
-		photo.setTags(photoIDs, newTags);
+		photo.setTags(photoIDs, newTags, data.override);
 	};
 
 	const setTagDialogBody = `
 		<p></p>
 		<form>
 			<div class="input-group stacked"><input class='text' name='tags' type='text' minlength='1'></div>
+			<div class='input-group compact-inverse'>
+				<label for="override">${lychee.locale["OVERRIDE"]}</label>
+				<input type='checkbox' id='override' name='override' />
```

Comment:
Having a short `name` for the input element is fine, because it is only used in the context of this method. However, `id` must be unique across the entire HTML page at any moment, and I fear that an ID `override` runs the risk of not being unique.

I had the problem with the ID `header` once, because an item inside our included SVG used the same ID and suddenly a lot of funny and seemingly unexplainable things were happening.

That is why I used rather long and clumsy IDs at other places like `id='tag_dialog_override_input'`, because that somewhat assures that nothing else used the same ID. For example, we include the Dropbox JS module. Can we guarantee that this does not accidentally contain an element whose ID is also `override`, because you might override files in a dropbox? Unfortunately, the HTML name space is flat.

Luckily, these long IDs are not really needed anywhere anymore, hence it doesn't matter how clumsy they are.